tests/Maintenance/MaintenanceTest.cpp

```diff
@@ -340,6 +381,7 @@ TEST_F(MaintenanceTestActionDescription, retrieve_nonassigned_key_from_actiondes
     auto bogus = desc.get("bogus");
     ASSERT_TRUE(bogus == "bogus");
   } catch (std::out_of_range const& e) {
+    e;
```

Comment:
exception variable name could be just omitted if we are not interested in exception details